Python/Product/Attacher/PyAttacher.cs

```diff
@@ -299,6 +306,23 @@ private static Process ConfigureAndStartProcess(ProcessStartInfo psi) {
         internal static DebugAttach AttachAD7Worker(int pid, int portNum, Guid debugId, string debugOptions, EventWaitHandle attachDoneEvent = null) {
             var hProcess = NativeMethods.OpenProcess(ProcessAccessFlags.All, false, pid);
             if (hProcess != IntPtr.Zero) {
+                try {
```

Comment:
I may be remembering wrong, but this method of attach isn't used anymore. This is the old ad7 stuff. If you follow this up through references, it's not used.